.gitignore

```diff
@@ -25,3 +25,7 @@ target
 
 .idea
 *.iml
+
+
+.history
+.vsocde
```

Comment:
Typo here

Also, please add a [newline at the end of the file](https://thoughtbot.com/blog/no-newline-at-end-of-file)